tests/codex/testerasure.nim

```diff
@@ -252,3 +252,22 @@ suite "Erasure encode/decode":
       decoded.treeCid == manifest.treeCid
       decoded.treeCid == verifiable.originalTreeCid
       decoded.blocksCount == verifiable.originalBlocksCount
+
+  for i in 1..5:
+    test "Should encode/decode using various parameters " & $i & "/5":
+      let
+        blockSize   = rng.sample(@[1, 2, 4, 8, 16, 32, 64].mapIt(it.KiBs))
```

Comment:
I think blocksizes of anything other than 64kb will completely break the proving system. It's definitely OK to pin this one.

Reply:
Ok, current test suite uses hardcoded 1kb block size. I think we should test at least a few different combinations in this unit test to make sure that EC works regardless of the block size.

Reply:
Yes, the circuit is hardcoded for 64k blocks (we hash 2kb cells, then build a depth 5 Merkle tree to get the block hash. This was done so that the proving system is "compatible" with network-level structure. We then prove 2kb cells against the slot hash.